### ISSUE TITLE
No user feedback when on wrong network

### ISSUE THREAD

**Issue by wanderingstan** [OriginProtocol/demo-dapp#172]
When user is on a network without our contracts the dapp silently fails. This happens on rinkeby when runnign local chain, or on `main` net. 

I know we used to pop up a message for this, but must have gotten lost somewhere. 

Chrome console shows: 
```
Uncaught (in promise) Error: This contract object doesn't have address set yet, please set an address first.
    at Object._processExecuteArguments (index.js:739)
    at Object._executeMethod (index.js:760)
    at Users._callee3$ (users.js:17)
    at tryCatch (runtime.js:65)
    at Generator.invoke [as _invoke] (runtime.js:303)
    at Generator.prototype.(:3000/anonymous function) [as next] (http://localhost:3000/app.9db4ddaf.js:46159:21)
    at step (users.js:1)
    at users.js:1
```

**Comment by micahalcorn** [comment]
@wanderingstan this came up a day or two ago and Nick reminded me that the modal only pops up in "production". We do need a fix so that an exception is not raised and the friendly Bootstrap-style alert is shown indicating that no contracts are found.

**Comment by nick** [comment]
Fixed in #173
